superset-frontend/src/components/EmptyState/index.tsx

```diff
@@ -36,7 +35,9 @@ import EmptySqlChartImage from 'src/assets/images/empty_sql_chart.svg';
 import EmptyQueryImage from 'src/assets/images/empty-query.svg';
 import EmptyTableImage from 'src/assets/images/empty-table.svg';
 import EmptyImage from 'src/assets/images/empty.svg';
+import { Button } from '../Button';
```

Comment:
There are multiple places where we use relative paths instead of `src/components`. Is that intentional or are those auto generated by your IDE?

Reply:
There have been some issues with circular dependencies caused by pulling components from the barrel file within components inside components and I figured we should just use relative imports. We might revisit this when we move to a dedicated package and split things a bit better. This has grown too large already.